packages/genesys-spark/src/index.ts

```diff
@@ -1,22 +1,29 @@
-import { getAssetsOrigin, getFontOrigin } from './hosts';
+import {
+  getComponentAssetsOrigin,
+  getChartComponentAssetsOrigin,
+  getFontOrigin
+} from './hosts';
 import {
   checkAndLoadScript,
   checkAndLoadStyle,
   checkAndLoadFonts
 } from './loading';
 
-const ASSET_PREFIX = '__ASSET_PREFIX__';
-const SCRIPT_PATH = 'genesys-webcomponents.esm.js';
-const STYLE_PATH = 'genesys-webcomponents.css';
+const COMPONENT_ASSET_PREFIX = '__COMPONENT_ASSET_PREFIX__';
+const CHART_COMPONENT_ASSET_PREFIX = '__CHART_COMPONENT_ASSET_PREFIX__';
 
-const assetsOrigin = getAssetsOrigin();
-const SCRIPT_SRC = `${assetsOrigin}${ASSET_PREFIX}${SCRIPT_PATH}`;
-const STYLE_HREF = `${assetsOrigin}${ASSET_PREFIX}${STYLE_PATH}`;
-const fontOrigin = getFontOrigin();
-const FONTS = {
-  Urbanist: `${fontOrigin}/webfonts/urbanist.css`,
-  'Noto Sans': `${fontOrigin}/webfonts/noto-sans.css`
-};
+/**
+ * TODO
+ */
+export function loadSparkFonts(): Promise<void> {
+  const fontOrigin = getFontOrigin();
+  const FONTS = {
+    Urbanist: `${fontOrigin}/webfonts/urbanist.css`,
```

Comment:
Should `Urbanist` be wrapped in single quotes ?

Reply:
no there is no space in it.